### PR TITLE
bws per wallet

### PR DESCRIPTION
**Pull request by javierbitpay**

### CODE REVIEW

**Comment by javierbitpay** [comment]
fixes #3221

File: src/js/controllers/join.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 'use strict';
 
 angular.module('copayApp.controllers').controller('joinController',
-  function($scope, $rootScope, $timeout, go, notification, profileService, isCordova, $modal, gettext, lodash, ledger, trezor) {
+  function($scope, $rootScope, $timeout, go, notification, profileService, configService, isCordova, $modal, gettext, lodash, ledger, trezor) {
 
     var self = this;
+    var defaults = configService.getDefaults();
+    var defaults = configService.getDefaults();
```
Comment (isocolsky) [inline src/js/controllers/join.js]: duplicated code

File: src/js/controllers/import.js [hunk 2]
```diff
@@ -22,6 +23,17 @@ angular.module('copayApp.controllers').controller('importController',
...
     };
 
+    var setBwsURL = function(walletId, cb) {
+      var opts = {
+        bws: {}
+      };
+      opts.bws[walletId] = $scope.bwsurl;
+      configService.set(opts, function(err) {
+        if (err) return cb(err);
+        return cb(null);
```
Comment (isocolsky) [inline src/js/controllers/import.js]: just pass the `cb` arg to the `configService.set` method

**Comment by cmgustavo** [comment]
LGTM. Great work!

**Pull request merged**